sdcm/provision/scylla_yaml/cluster_builder.py

```diff
@@ -171,7 +167,7 @@ def _open_ldap_bind_passwd(self) -> str:
 
     @property
     def _open_ldap_server_address_port(self) -> str:
-        if self.params.get('ip_ssh_connections') == 'public':
+        if self.test_config.IP_SSH_CONNECTIONS == 'public':
```

Comment:
Why I need to call again the function that already was run and not use the ready result?

Reply:
I think we might want to remove IP_SSH_CONNECTIONS completely at some point, so introducing usages of it, doesn't really help moving in that direction.

from my POV it's a nitpick, since at the end, once we support this for all backends, we will remove it.